src/hermes.jsx

```diff
@@ -305,6 +307,7 @@ module.exports = React.createClass({
 	},
 
 	onBlur: function () {
+		this.clearSuggestions();
```

Comment:
I am not sure this is the behavior we want.  Does it not loose focus when you select a suggestion with your keyboard?  Like tabbing down?